Leanplum-SDK/Classes/Utils.m

```diff
@@ -29,9 +29,13 @@ @implementation Utils
 
 + (BOOL)isNullOrEmpty:(id)obj
 {
+    // Need to check for NSString to support RubyMotion.
+    // Ruby String has count: method and respondsToSelector(count) is true for count:
+    // but count: in Ruby requires parameters
```

Comment:
Almost.
count: in general requires parameters. This is same for Objective-c and is not an issue. 

The issue is that normally respondsToSelector(count:) is not same as respondsToSelector(count).
However in RubyMotion they are the same.

Example:
If the object has count: but does not have count it should
- respondsToSelector(count) = false
- respondsToSelector(count:) = true

However RubyMotion deliberately says that it is true. So in the same scenario it will be
- respondsToSelector(count) = true
- respondsToSelector(count:) = true

I will change to
respondsToSelector(count) is true for count: in RubyMotion
and remove // but count: in Ruby requires parameters